vyper/ast/nodes.py

```diff
@@ -405,72 +404,56 @@ class ClassDef(VyperNode):
 
 class Constant(VyperNode):
     # inherited class for all simple constant node types
-    __slots__ = ()
+    __slots__ = ('value', )
 
 
 class Num(Constant):
     # inherited class for all numeric constant node types
-    __slots__ = ('n', )
-    _translated_fields = {'value': 'n'}
+    __slots__ = ()
     _python_ast_type = "Num"
+    _translated_fields = {'n': 'value'}
+
+    @property
+    def n(self):
```

Comment:
How annoying would it be to have these throw Warnings?

Reply:
50 places in the code or so annoying lol

Reply:
At least add a TODO

Reply:
Also for `s`